src/components/pages/ImageClassification/ImageAnnotationModal/ImageAnnotationPopup/SelectAttributeFromClassifierGuesses.js

```diff
@@ -111,18 +156,82 @@ const ExistingRows = ({
                 {row.label}
               </option>
             ))}
+            <option disabled>──────────</option>
```

Comment:
I really want to use an `<hr>` which is now semantically legal, but a few mobile browsers dont support it still and we probably need to support large tablets. Im just going with the old hack for now and look forward to a couple of years in the future when we can just be less hacky. :) 

(I havent run into this issue for a really long time mostly using component libraries in the recent past which hid this issue, so I almost didnt believe it was necessary!)